src/snyk/cli/messages/messages.ts

```diff
@@ -1,14 +1,15 @@
 export const messages = {
-  progressTitle: 'Downloading Snyk CLI...',
-  startingDownload: 'Starting Snyk CLI download...',
-  startingUpdate: 'Starting Snyk CLI update...',
-  isLatest: 'Snyk CLI version is up-to-date.',
-  notSupported: 'Snyk CLI cannot be downloaded because OS platform is not supported.',
-  integrityCheckFailed: 'The downloaded Snyk CLI integrity check failed.',
-  couldNotDeleteExecutable: 'Could not delete existing CLI executable.',
-  downloadCanceled: 'CLI download has been canceled.',
-  cliDownloadFailed: 'Failed to download or update Snyk CLI.',
+  progressTitle: 'Downloading Snyk plugin dependencies...',
```

Comment:
Can we ditch the "plugin" word completely? "Extension" would be the correct term here but imo it doesn't provide any value to the user.

Reply:
no, meant more like:
> Downloading Snyk dependencies

> Snyk dependencies download has been canceled.


etc

Reply:
oh, yeah, you're correct. Misread the msg